src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -991,17 +1047,21 @@ public void TypeDeclaration_NoInstanceInitializers()
     public int Property { get; set; } = 43;
 }
 ";
-            // This will be allowed in C# 10, or we need to improve the message
-            // Tracking issue https://github.com/dotnet/roslyn/issues/52240
```

Comment:
Does this PR close [https://github.com/dotnet/roslyn/issues/52240](https://github.com/dotnet/roslyn/issues/52240) ?